### ISSUE TITLE
Update heuristics

### ISSUE THREAD

**Issue by pondering** [dreal/dreal#71]
Since we modify the internal representation of hybrid model, the `heuristics` module needs to update as well.

**Comment by danbryce** [comment]
Yes, I will do this sometime soon.  Thanks for generalizing to multiple jumps between pairs of modes.  Is there a simple benchmark that has this?

Thanks,

Dan
